src/lib.rs

```diff
@@ -5,9 +5,13 @@ pub type LineId = usize;
 
 pub type LcsToken = String;
 pub type LcsSeq = Vec<LcsToken>;
+pub type LcsDelimiters = Vec<char>;
 
-pub fn tokenize(input: &str) -> impl Iterator<Item = &str> {
-    input.trim().split_whitespace().filter(|s| !s.is_empty())
+/** 
+ * Tokenizer is using delimiters specified in LcsMap::new to create the tokens.
+*/
```

Comment:
Can you replace the comment style to use markdown? That would make it like this:
```suggestion
/// Tokenizer is using delimiters specified in LcsMap::new to create the tokens.
```

See [Chapter 14-2](https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html#making-useful-documentation-comments) of the book.